Blockzilla/metrics.yaml

```
@@ -59,6 +59,19 @@ mozilla_products:
     expires: "2022-09-01"
 
 settings_screen:
+  set_as_default_browser_pressed:
+    type: counter
+    description: |
+      Counts the number of times setting as default
+      browser menu option is tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-ios/issues/3328
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-ios/pull/3344
+    notification_emails:
+      - focus-ios-data-stewards@mozilla.com
+    expires: "2023-07-20"
```

Please add a `data_sensitivity` to this metric, it helps make this easier to see in the Glean Dictionary, etc.

```suggestion
    data_sensitivity:
      - interaction
    expires: "2023-07-20"
```

---

yes, I will add this change. Thank you!